### ISSUE TITLE
Additional Exporting Features

### ISSUE THREAD

**Issue by GameDotExe** [ClassicLootManager/ClassicLootManager#304]
When I export data to JSON or XML output data is limited compared to what is available in display in addon GUI. Some categories not exported include decay loss per reduction in exact amount, total decay for period requested, decay as a % instead of a whole number (reports as 25 instead of 25% or 0.25), raid attendance % per ID, raid attendance for period requested, weekly gains per ID, total spent per ID, total received per ID, total blocked per ID, total decayed per ID. Would also like to export as a CSV file in addition to XML and JSON.

It would be helpful to have this information included with exported data. It would be convenient for the exported data to be in CSV.

Not many alternative options exist to the suggestion.

Thank you for the consideration and hard work.

**Comment by msuter-cbre** [comment]
I agree with the request for adding a CSV export option. Our GM would like to be able to export data for specific raids as well, so if a method was added to export based on Raid that would be really appreciated!
